polyaxon/templates/secrets.yaml

```diff
@@ -77,3 +77,20 @@ data:
   postgres-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{ end }}
 {{- end -}}
+---
+{{ if (index .Values "docker-registry").auth.password }}
```

Comment:
@mouradmourafiq I disagree with this. The way I see this is that auth is separate feature (that can be enabled). If you check the `values.yaml` I separated the `user` and `password` from `external` key and created a `auth` key.

So that's why I don't check `install` like you do on postgresql.

But let me know if you don't agree with this.